modules/world/evaluation/evaluator_right_overtake.cpp

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) 2020 fortiss GmbH
+//
+// This work is licensed under the terms of the MIT license.
+// For a copy, see <https://opensource.org/licenses/MIT>.
+
+#include "modules/world/evaluation/evaluator_right_overtake.hpp"
+
+#include "modules/world/evaluation/labels/behind_of_label_function.hpp"
+#include "modules/world/evaluation/labels/dense_traffic_label_function.hpp"
+#include "modules/world/evaluation/labels/front_of_label_function.hpp"
+#include "modules/world/evaluation/labels/right_of_label_function.hpp"
+
+namespace modules {
+namespace world {
+namespace evaluation {
+const char EvaluatorRightOvertake::formula_[] =
+    "G (!dense -> !(b_v#0 & X[!](b_v#0 U r_v#0 U f_v#0)))";
```

Comment:
add what this means

Reply:
rename predicates so they are congruent with the conventions in the other labels